app/schemas.py

```diff
@@ -58,6 +59,18 @@ def make_instance(self, data):
 
 
 class UserSchema(BaseSchema):
+
+    permissions = fields.Method("user_permissions", dump_only=True)
+
+    def user_permissions(self, usr):
```

Comment:
Can this be done with a join? So that the user object always has permissions? Sorry if I'm missing some marshmallow stuff here that does this.